src/CoreAPIClient.ts

```diff
@@ -81,4 +86,22 @@ export default class CoreAPIClient {
                 })
             })
     }
+
+    private async _get(apiUrl: string): Promise<any> {
```

Comment:
Methods for POST and GET should return the same type to make the API uniform and easy to understand.